src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -11,20 +22,55 @@
 import java.util.function.DoubleSupplier;
 
 public class Climber extends SubsystemBase implements AutoCloseable {
-  private final DoubleSolenoid anglerPiston =
+  private final DoubleSolenoid piston =
```

Comment:
can you revert this rename? it obfuscates the git diff, making it hard to see what actual changes were made and what changes are due to this rename. if you want, open a separate PR that renames the field